cmd/server/main.go

```diff
@@ -1,31 +1,48 @@
 package main
 
 import (
-	"fmt"
 	"log"
 	"net/http"
+	"time"
 
 	"evgen3000/go-musthave-metrics-tpl.git/cmd/server/router"
 	"evgen3000/go-musthave-metrics-tpl.git/cmd/server/storage"
-	"evgen3000/go-musthave-metrics-tpl.git/internal/config"
+	"evgen3000/go-musthave-metrics-tpl.git/internal/config/server"
+	httpLogger "evgen3000/go-musthave-metrics-tpl.git/internal/logger"
 	"github.com/go-chi/chi/v5"
+	"go.uber.org/zap"
 )
 
-func runServer(host string, router *chi.Mux) {
-	fmt.Println("Server is running on", host)
-	err := http.ListenAndServe(host, router)
+func runServer(config *server.Config, router *chi.Mux) {
+	logger := httpLogger.InitLogger()
+	logger.Info("server is running on", zap.String("host", config.Host))
+	err := http.ListenAndServe(config.Host, router)
 	if err != nil {
-		log.Fatal(err)
+		logger.Fatal("Error", zap.String("Error", err.Error()))
 	}
 }
 
 func main() {
-	c := config.GetHost()
-	s := storage.NewMemStorage()
+	c := server.GetServerConfig()
+	s := storage.NewMemStorage(storage.MemStorageConfig{
+		StoreInterval:   c.StoreInterval,
+		FileStoragePath: c.FilePath,
+		Restore:         c.Restore,
+	})
+
 	r := router.SetupRouter(s)
 
-	runServer(c.Value, r)
-}
+	ticker := time.NewTicker(time.Duration(c.StoreInterval) * time.Second)
```

Comment:
```suggestion
	ticker := time.NewTicker(c.StoreInterval)
```
[CHANGE REQUEST] It should be `Duration`